apps/docs/.vitepress/config.ts

```diff
@@ -361,10 +361,6 @@ export default defineConfig({
             link: '/guide/cookbook/',
             collapsed: true,
             items: [
-              {
-                text: 'Transferring Assets',
-                link: '/guide/cookbook/transferring-assets',
```

Comment:
We need to delete this file: `apps/docs/src/guide/cookbook/transferring-assets.md`, as it is not going to be used anymore.